doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
+  CentOS Linux distribution is not supported anymore.
```

Comment:
There will be no need to mention any other specific distributions when we have already specified exactly what is supported. We can remove this statement.

Reply:
Done. Thank you.